src/tsickle.ts

```diff
@@ -2130,6 +2130,23 @@ export function emitWithTsickle(
   };
 }
 
+function isSymbolDeclaredInSourceFile(symbol: ts.Symbol, sourceFile: ts.SourceFile) {
+  const declarations = symbol.getDeclarations();
+  if (!declarations) {
+    // If there are no declarations, err on the side of safety and emit the clutz
+    // aliases
+    return true;
+  }
+
+  for (const decl of declarations) {
+    if (decl.getSourceFile() === sourceFile) {
+      return true;
+    }
+  }
+
+  return false;
+}
+
 function addClutzAliases(
```

Comment:
I have no idea what this function is supposed to do, heh.

Reply:
Added a comment - does it make sense?